src/components/application_manager/rpc_plugins/sdl_rpc_plugin/include/sdl_rpc_plugin/commands/hmi/navi_audio_start_stream_request.h

```diff
@@ -77,13 +77,19 @@ class AudioStartStreamRequest : public app_mngr::commands::RequestToHMI,
    **/
   virtual void on_event(const app_mngr::event_engine::Event& event);
 
+ private:
   /**
    * @brief RetryStartSession resend HMI startSession request if needed.
    * If limit expired, set audio_stream_retry_number counter to 0
```

Comment:
@AKalinich-Luxoft as a suggestion -
we can specify in the description that `RetryStartSessionInternal` function will be called inside and why do we need it

Reply:
@LitvinenkoIra I thought about that...
Actually, I decided to remove that function to avoid confusions just because `RetryStartSessionInternal` is in the same namespace and with the same visibility so it might be called accidentally when it shouldn't. I encapsulated this logic inside of lambda inside of `RetryStartSession` to make code cleaner.
Please double check that commit - https://github.com/smartdevicelink/sdl_core/pull/3552/commits/d23a8851f67ef834f9e40be39e2c9c5b17b55664